backend/controllers/page_controller.py

```diff
@@ -55,7 +55,7 @@ def create_page(page: PageCreate, db: Session = Depends(get_db)):
     return db_page
 
 
-@PAGE_CONTROLLER.get("/", response_model=list[PageOut]) #TODO add pagination on the frontend
+@PAGE_CONTROLLER.get("/", response_model=list[PageOut]) # TODO: add pagination on the frontend
 def read_all_pages(
```

Comment:
Ja by som to spravil tak aby to nebralo `skip` a `limit`, ale `page` a `per_page` (`page` bude aktuálna stránka, `per_page` bude počet záznamov na stránku).

Taktiež ošetri prípad, keď používateľ zadá `limit` (alebo `per_page`, ak sa to rozhodneš spraviť tým spôsobom) ktorý je napríklad 10000, potom backend vráti 10000 záznamov a nemá to zmysel optimalizovať touto cestou, keďže každý si môže cez API priamo vyžiadať milión záznamov